docs/DevGuide/Travis.md

```diff
@@ -74,6 +74,29 @@ performed in the CHECK_FILES build.
 using a Linux OS, and the `AppleClang` compiler for `OS X`.
 * The `gcc Debug` build will fail if there are `doxygen` warnings.
 
+## How to perform the checks locally
+
+Before pushing to GitHub and waiting for Travis to perform the checks it is
+useful to perform at least the following tests locally:
+- **Unit tests:** Perform a `make test-executables` and then `ctest -L unit` to
+  run all unit tests. This can take a significant amount of time, so to run only
```

Comment:
Remove the "can take a significant amount of time". Our tests run really, really fast compare to any other project I've seen of this size :)